srv/components/csm/prepare.sls

```diff
@@ -36,13 +36,13 @@ Add CSM repo:
     - gpgkey: {{ defaults.csm.repo.gpgkey }}
 {% endif %}
 
-{% if 'single' not in pillar['cluster']['type'] %}
-Render CSM ha input params template:
-  file.managed:
-    - name: /opt/seagate/cortx/ha/conf/build-ha-csm-args.yaml
-    - source: salt://components/csm/files/ha-params.tmpl
-    - template: jinja
-    - mode: 444
-    - makedirs: True
-{% endif %}
+#{% if 'single' not in pillar['cluster']['type'] %}
```

Comment:
Jinja comments are different.
This would not work.

Ref:
https://stackoverflow.com/a/51201182